### ISSUE TITLE
Fix generated code for defaults

### ISSUE THREAD

**Issue by dmoisset** [machinalis/protobuf-python3#20]
The generated code includes lots of lines saying

`default_value=unicode("...", "utf-8")`

which probably should be b"...".decode("utf-8")

**Comment by dmoisset** [comment]
Fixed in db0321c.

Note that to test this you should delete all generated *_pb2.py files, to Force regeneration. Also you should recompile the protoc. There's no need to install the compiler, the test script uses the locally compiled version if available.

**Comment by ramiro** [comment]
What I had to do:

```
$ cd protobuf-python3
$ ./autogen.sh    # generates the `configure` script
$ ./configure
$ make
```

The above process leaves our updated `protoc` binary in the `src` subdir.

```
$ make check # this runs the protobuf test suite
```

Remove the *_pb2.py files:

```
$ cd python
$ find . -type f -name "*_pb2.py" -delete
```

Now we can trigger the regeneration of these files e.g. by running the tests. Make sure to have our Python virtualenv active:

```
$ python -3 setup.py test  # This re-creates the *_pb2.py files with the `protoc` located in `../src/`
```

Just in case, check that we don't have any `default_value=unicode("...", "utf-8")` in the generated Python code anymore:

```
$ ack unicode --py
```
